web/src/containers/Admin/User/AboutData.js

```diff
@@ -29,6 +29,7 @@ import Image from 'components/Image';
 import withConfig from 'components/ConfigProvider/withConfig';
 
 const VerificationForm = AdminHocForm('VERIFICATION_FORM');
+const MANUAL_KYC_PLUGINS = ['kyc', 'kyc-mongol'];
```

Comment:
This is not good. @brandonkimmmm and I decided to make these uniform so we don't need to have such logic @ahslr